builder/src/scss/package-list.scss

```diff
@@ -1,5 +1,121 @@
 @import "./theme/_variables";
 
+.lander-column-margins {
+    margin-top: 8%;
+    margin-bottom: 8%;
+}
+
+.mod-count-box > img {
+    max-width: 96px;
+    max-height: 96px;
+}
+
+.modspic {
+    max-width: 500px;
+    max-height: 500px;
+}
+
+.mod-manager-pic {
+    width: 12rem;
+    height: 12rem;
+}
+
+.mod-man-banner {
+    max-width: 60%;
+    max-height: 15rem;
+    margin: 2rem auto 5rem auto;
+    position: relative;
+}
+
+.mod-count-box > p {
+    margin-bottom: 0;
+}
+
+.ts-primary-button {
+    font-size: 14px;
+    padding-left: 1rem;
+    padding-right: 1rem;
+    padding-top: 0.5rem;
+    padding-bottom: 0.5rem;
+    font-weight: bold;
+    background-color: #00d2ff;
+    color: #000001;
+}
+
+.ts-secondary-button {
+    font-size: 14px;
+    padding-left: 3%;
+    padding-right: 3%;
+    padding-top: 1%;
+    padding-bottom: 1%;
+    font-weight: bold;
+    background-color: #00000100;
+    border-width: 0;
+}
+
+.ts-card {
+    position: relative;
+    font-size: 16px;
+    overflow: hidden;
+    width: 12rem;
+    height: 20rem;
+    flex-grow: 1;
+    border-radius: 0.4rem !important;
+    object-fit: cover;
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center;
+}
+
+.ts-card-bg {
+    border-radius: 0.4rem !important;
+    height: 100%;
+}
+
+.ts-card-content {
+    position: absolute;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    height: 100%;
+    justify-content: flex-end;
+    align-items: center;
+    padding: 5em 2em 2em 2em;
+    background: linear-gradient(
+        0deg,
+        rgba(0, 0, 0, 1) 0%,
+        rgba(0, 0, 0, 0.5) 20%,
+        rgba(0, 0, 0, 0.3) 30%,
+        rgba(0, 0, 0, 0) 60%
+    );
+}
+
+.community-list-footer {
+    background: rgb(15, 15, 15);
+    border-top-left-radius: 0.4rem !important;
+    border-top-right-radius: 0.4rem !important;
+    border-bottom-left-radius: 0.4rem !important;
+    border-bottom-right-radius: 0.4rem !important;
+    padding: 1.5rem 0.5rem 1.5rem 0.5rem;
+    margin-bottom: 6rem;
+}
+
+.ts-rounded {
+    border-radius: 0.4rem !important;
+}
+
+.footer-item {
+    font-size: 14px;
+}
+
+.ts-text-highlight {
+    color: #2f8ec4;
+}
+
+.ts-card > div > a > h5 {
+    color: white !important;
```

Comment:
`!important` should'nt be needed.

Also this is not a blocker and kind of comes down to personal preferences, or more as to what we collectively agree upon. My preference is to never use css color names, and opt using hex/rgb(a)/hsl values instead. This is not so much of an issue with `white`, because we can easily deduct the hex value for it would be `#fff`, but I for one can't remember the hex value for e.g. `lavender`. Thus, for consistency and legibility, I recommend always using hex/rgb(a)/hsl values.